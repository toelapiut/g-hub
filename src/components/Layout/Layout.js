import styles from './styles.module.css';
import PropTypes from 'prop-types';


export const Layout = ({children}) => {
  return (
    <div className={styles.contain}>
      {children}
    </div>
  );
};

Layout.propTypes = {
  children: PropTypes.oneOfType([
    PropTypes.object,
    PropTypes.func
  ]),
};