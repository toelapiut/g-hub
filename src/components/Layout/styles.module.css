.contain{
    max-width: 80rem;
    @apply m-auto bg-night;
}