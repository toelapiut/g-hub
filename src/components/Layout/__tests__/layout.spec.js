import React from 'react';
import Layout from '../';
import renderer from 'react-test-renderer';
import {describe, it} from '@jest/globals';

describe('layout Component', () => {

  it('matches the navbar snapshots', () => {
    const tree = renderer.create(
      <Layout/>
    ).toJSON();
    expect(tree).toMatchSnapshot();
  });
});