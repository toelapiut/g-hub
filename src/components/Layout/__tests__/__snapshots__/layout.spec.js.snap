// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`layout Component matches the navbar snapshots 1`] = `
<div
  className="contain"
/>
`;
